scripts/start/submit.R

```diff
@@ -49,39 +49,38 @@ submit <- function(cfg, restart = FALSE, stopOnFolderCreateError = TRUE) {
       # detected like this
       firstRunInCascade <- normalizePath(renv::project()) == normalizePath(".")
       if (firstRunInCascade) {
-        capture.output({ # suppress printing status
-          renvStatus <- renv::status()
-        })
-        if (!renvStatus$synchronized) {
-          message("The new run will use the package environment defined in renv.lock, ",
-                  "but it is out of sync, probably because you installed packages/updates manually. ",
-                  "Write current package environment into renv.lock first? (Y/n)", appendLF = FALSE)
-          if (tolower(gms::getLine()) %in% c("y", "")) {
-            renv::snapshot(prompt = FALSE)
-          }
-        }
-
         if (getOption("autoRenvUpdates", FALSE)) {
-          source("scripts/utils/updateRenv.R")
+          installedUpdates <- piamenv::updateRenv()
+          piamenv::stopIfLoaded(names(installedUpdates))
         } else if (   'TRUE' != Sys.getenv('ignoreRenvUpdates')
                    && !is.null(piamenv::showUpdates())) {
-          message("Consider updating with `Rscript scripts/utils/updateRenv.R`.")
+          message("Consider updating with `piamenv::updateRenv()`.")
```

Comment:
```suggestion
          message("Consider updating with `make update-renv`.")
```
I'd like to put something there which can be copy+pasted verbatim to the comand line.